dash/dash-renderer/src/serializers/utils.js

```diff
@@ -0,0 +1,40 @@
+import {type, prop} from 'ramda';
+
+const PROP_TYPE = '__type';
+const PROP_VALUE = '__value';
+export const DASH_BOOK_KEEPER = '__dash_serialized_props';
+
+export const createBookkeeper = layout => {
+    const markedLayout = {...layout, __dash_serialized_props: {}};
+    const {
+        props,
+        props: {children}
+    } = layout;
+    if (type(children) === 'Array') {
+        markedLayout.props.children = children.map(child =>
+            createBookkeeper(child)
+        );
+    }
+
+    if (type(children) === 'Object') {
+        createBookkeeper(children);
```

Comment:
`createBookkeeper()` returns a modified Object here, shouldn't we update markedLayout's value with the returned value?